apps/wing/src/cli.ts

```diff
@@ -49,8 +49,9 @@ async function main() {
   program
     .command("run")
     .alias("it")
-    .description("Runs a Wing simulator file in the Wing Console")
-    .argument("[simfile]", ".wsim simulator file")
+    .description("Runs a Wing program in the Wing Console")
+    .argument("[entrypoint]", "program .w entrypoint")
```

Comment:
Would it be possible for this to work with both a .w file and .wsim dir? If not, I think it's fine to just work with a .w IMO, but I can see the usefulness of being able to open a .wsim without compilation.

Reply:
It's technically possible but we currently assume it's a wingfile that needs compilation. We could make the distinction based on the extension and skip the compilation if necessary, but I'd suggest doing it in another issue.